src/ontology/cl-edit.owl

```diff
@@ -32633,6 +32636,14 @@ AnnotationAssertion(terms:date obo:CL_4047019 "2024-09-24T12:11:09Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047019 "early enterocyte"@en)
 EquivalentClasses(obo:CL_4047019 ObjectIntersectionOf(obo:CL_0000584 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0013740)))
 
+# Class: obo:CL_4047025 (type G cell of stomach)
+
+AnnotationAssertion(terms:contributor obo:CL_4047025 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047025 "2024-10-24T10:43:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4047025 "type G cell of stomach"@en)
+EquivalentClasses(obo:CL_4047025 ObjectUnionOf(ObjectIntersectionOf(obo:CL_0000508 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001276)) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004997)))
```

Comment:
This logical definition is not correct - I believe we can't use **part of this OR part of this** 
```
('type G enteroendocrine cell'
 and ('part of' some 'epithelium of stomach')) or ('part of' some 'mucosa of pyloric antrum').
```
In my previous comment, I meant you can use
```
 ('type G enteroendocrine cell'
 and ('part of' some 'epithelium of stomach'))
```

OR

```
('type G enteroendocrine cell'
 and ('part of' some 'mucosa of pyloric antrum')).
```

Depends on how specific you want it to be.

Looking more into literature, It seems to me that` part of some pyloric gland `- which is a synonym for pyloric antrum glands might be more suitable but it is up to you. Epithelium of stomach would also work.
<img width="358" alt="image" src="https://github.com/user-attachments/assets/8a046bb7-5e57-4501-8369-fc4a1ec58382">


 I found this 

(G cells are found deep within the [pyloric glands](https://en.wikipedia.org/wiki/Pyloric_glands) of the [stomach](https://en.wikipedia.org/wiki/Stomach) [antrum](https://en.wikipedia.org/wiki/Pyloric_antrum).) https://en.wikipedia.org/wiki/G_cell

Reply:
epithelium of stomach would also work.. It depends how specific you want it to be

Reply:
Amended to first option only